iASLUITests/MonkeyTests.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  MonkeyTests.swift
+//  iASLUITests
+//
+//  Created by Ian Applebaum on 4/17/20.
+//  Copyright © 2020 Y Media Labs. All rights reserved.
+//
+
+import XCTest
+import SwiftMonkey
+class MonkeyTests: XCTestCase {
+	let application = XCUIApplication()
+	override func setUp() {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+
+        // UI tests must launch the application that they test. Doing this in setup will make sure it happens for each test method.
+         // Put setup code here. This method is called before the invocation of each test method in the class.
+				application.launchArguments.append("ui-testing")
+		application.launchArguments.append("monkeyTesting")
+				// In UI tests it is usually best to stop immediately when a failure occurs.
+				continueAfterFailure = false
+				// UI tests must launch the application that they test. Doing this in setup will make sure it happens for each test method.
+				application.launch()
+				
+				
+
+        // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
+    }
+
+	override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+	func testMonkeyFromLogin() {
+
+	//		application.launchArguments = ["--MonkeyPaws"]
+			
+			addUIInterruptionMonitor(withDescription: "System Dialog") { (alert) -> Bool in
+				  let okButton = alert.buttons["OK"]
+				  if okButton.exists {
+					okButton.tap()
+				  }
+
+				  let allowButton = alert.buttons["Allow"]
+				  if allowButton.exists {
+					allowButton.tap()
+				  }
+
+				  return true
+				}
```

Comment:
@likhongomes @pucksindeep So this line is supposed to address those dumb pop ups (Speech privacy and camera privacy) with a fresh install of the app.